collect_app/src/main/res/values/buttons.xml

```diff
@@ -6,6 +6,15 @@
         <item name="android:textStyle">normal</item>
     </style>
 
+    <style name="Widget.Collect.Button.Layout" parent="Widget.Collect.Button.Custom">
```

Comment:
The naming scheme here is pretty confusing as by convention we use `Widget` at the beginning but that's referring to the general definition of the word rather than a "question widget". Maybe the name here could be `Widget.Collect.Button.WidgetAnswer`?

Reply:
Oh I thought about it initially.